Extensions/one_click_postage.js

```diff
@@ -657,20 +657,26 @@ XKit.extensions.one_click_postage = new Object({
 			XKit.storage.set("one_click_postage", "share_on_" + $(this).attr('data-site'), m_value);
 		});
 
-		$(document).on("mouseover", ".reblog_button,.post_control.reblog", function(event) {
+		const reblog_buttons = [
+			'.reblog_button',
+			'.post_control.reblog',
+			'button[aria-label="Reblog"]'
```

Comment:
is "Reblog" consistent across all languages?

Reply:
no but i'm going to add the translation function in a separate PR since i want to get this out tonight.